tests/e2e/operator.sh

```diff
@@ -84,8 +87,15 @@ install_ccruntime() {
 		fi
 	done
 
-	# TODO: check the runtime is up.
-	# kubectl get runtimeclass
+	# Check if the runtime is up.
+	# There could be a case where it is not even if the pods above are running.
+	cmd="kubectl get runtimeclass | grep -q ${runtimeclass}"
+	if ! wait_for_process 300 30 "$cmd"; then
+		echo "ERROR: runtimeclass ${runtimeclass} is not up"
+		return 1
+	fi
+	# To keep operator running, we should resume registry stopped during containerd restart.
+	start_local_registry
```

Comment:
Yep, we didn't have this problem when working with an image at `quay.io`, which is always available. What I've confirmed during the test is as follows:

1. operator is running. &rightarrow; moving to the installation of `ccruntime`
2. `preInstall` for `ccruntime` is triggered &rightarrow; containerd [restarts](https://github.com/confidential-containers/operator/blob/main/install/pre-install-payload/scripts/container-engine-for-cc-deploy.sh#L56)
3. A `manager` container of the operator pod restarts as well and fails to fetch the image because `local-registry` is stopped during the restart of containerd and not available.
4. Even if  the operator stays in `ImagePullBackOff`, this doesn't affect a test result up until those of operator (reinstall and install operator)
5. The test `Test can reinstall the operator` hangs due to the status of the operator.

So I had to put the comand to resume the local registry.

Reply:
Thanks for this detailed comment @BbolroC !